cl/opinion_page/views.py

```diff
@@ -293,8 +294,30 @@ def view_opinion(request, pk, _):
             'caller': 'view_opinion',
         }
         citing_clusters = conn.raw_query(**q).execute()
+
+        # Related opinions with Solr-MoreLikeThis query
+        # (Beta test: feature is only available for specific user groups)
+        if request.user.is_authenticated and (request.user.is_superuser or request.user.is_staff
+            or (hasattr(settings, 'RELATED_USER_GROUPS') and
+                request.user.groups.filter(name__in=settings.RELATED_USER_GROUPS).exists())):
+
+            # Use cache if enabled
+            mlt_cache_key = 'opinion-mlt%s' % pk
```

Comment:
I'd do a colon here, and I'd flip the namespacing so it's:

`mlt-opinion:32`

For example.

Reply:
Fixed.